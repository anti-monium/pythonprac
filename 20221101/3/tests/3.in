a = Grange(2, 2, 1000000000)
b = Grange(1, 9, 1001000100101001)
c = Grange(3, 14, 1456798)
print(a, b, c)
print(*list(a))
print(*list(b[7:8888:1]))
print(c[0], c[9])
print(bool(a) + bool(b) + bool(c))
print(c[99:10080])
