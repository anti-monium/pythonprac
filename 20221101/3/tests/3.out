grange(2, 2, 1000000000) grange(1, 9, 1001000100101001) grange(3, 14, 1456798)
2 4 8 16 32 64 128 256 512 1024 2048 4096 8192 16384 32768 65536 131072 262144 524288 1048576 2097152 4194304 8388608 16777216 33554432 67108864 134217728 268435456 536870912
7 63 567 5103
3 61983140352
3
grange(99, 14, 10080)
